src/interpolations.jl

```diff
@@ -489,6 +496,13 @@ function reference_coordinates(::Lagrange{RefLine,1})
             Vec{1, Float64}(( 1.0,))]
 end
 
+# Mapping from to 0D node to 1D line vertex.
+function transfer_point_face_to_cell(point::Vec{N, T}, cell::Type{RefLine}, face::Int) where {N, T}
```

Comment:
I think these methods belong in face_integrals.jl where the craete_face_quad_rule is

Reply:
Thanks for the review!
Will move them and push after CI finishes (making sure no older version break, had some issues with 1.6 before :"D)

Reply:
I'll push a commit for better errors/coverage after it passes locally, hopefully this fully covers the PR